org.eclipse.january/src/org/eclipse/january/dataset/Maths.java

```diff
@@ -350,10 +350,12 @@ public static Dataset abs(final Object a, final Dataset o) {
 			if (as == 1) {
 				while (it.hasNext()) {
 					oc64data[it.oIndex] = (float) Math.abs(it.aDouble);
+					oc64data[it.oIndex+1] = 0;
```

Comment:
The fix will need to be more general than this as the same issue will occur with other compound datasets in methods where the output has been given.